src/utils/calculateClicks.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+
+const calculateClicks = (campaignOrChannel, data) => {
```

Comment:
DRY with calculateClicks and calculateImpressions

merge those two into one

Reply:
a01fb507ddd48c3aaf56b9f518982ad20b70dde2